### PR TITLE
chore(core): json types in web console settings

### PR DESCRIPTION
**Pull request by glasstiger**
required by https://github.com/questdb/questdb-enterprise/pull/392

tandem, but can be merged without https://github.com/questdb/questdb-enterprise/pull/392

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 8 / 14 (57.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cutlass/http/processors/SettingsProcessor.java|8|14|57.14%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 8 / 14 (57.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cutlass/http/processors/SettingsProcessor.java|8|14|57.14%|

File: core/src/main/java/io/questdb/cutlass/http/processors/SettingsProcessor.java [hunk 1]
```diff
@@ -32,29 +32,47 @@
...
 import io.questdb.std.ObjList;
+import io.questdb.std.str.Utf8Sequence;
+import io.questdb.std.str.Utf8String;
 import io.questdb.std.str.Utf8StringSink;
+import io.questdb.std.str.Utf8s;
 
 import java.net.HttpURLConnection;
 
 public class SettingsProcessor implements HttpRequestProcessor {
+    private static final Utf8String V2 = new Utf8String("v2");
```
Comment (puzpuzpuz) [inline core/src/main/java/io/questdb/cutlass/http/processors/SettingsProcessor.java:45]: Do we need versioning? I thought this version is not yet released.

Comment (glasstiger) [reply core/src/main/java/io/questdb/cutlass/http/processors/SettingsProcessor.java:45]: good point, let me remove versioning

Comment (glasstiger) [reply core/src/main/java/io/questdb/cutlass/http/processors/SettingsProcessor.java]: done

**Review by puzpuzpuz — APPROVED** [review]

**Review by ideoma — DISMISSED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:disappointed: **fail** : 0 / 1 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cutlass/http/processors/SettingsProcessor.java|0|1|00.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 4 / 5 (80.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|3|4|75.00%|
|:large_blue_circle:|io/questdb/cutlass/http/processors/SettingsProcessor.java|1|1|100.00%|

**Review by ideoma — APPROVED** [review]

**Pull request merged**